backend/pkg/logger/errors.go

```diff
@@ -7,12 +7,14 @@ import (
 	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
 )
 
+// TODO! Check methods using value and point receivers for the same function
+
 type ErrLoggerNotFound struct {
 	Name abstraction.LoggerName
 }
 
 func (err ErrLoggerNotFound) Error() string {
-	return fmt.Sprintf("Logger %s not found", err.Name)
+	return fmt.Sprintf("Error %s logger not found", err.Name)
```

Comment:
since this is not where the error is actually printed out, I prefer to do something like this:
```go
fmt.Sprintf("logger %s not found", err.Name)
```
With no capitalization so that later on the printed out message looks more clean

Reply:
I am seeing you are doing this in other places, if you want to leave it like is I am ok with it, but if you change it here change it also in the rest of the code

Reply:
It's only my lingo preference to identify errors easily, I'll leave it like that and be consistent with it.